calico-enterprise/getting-started/install-on-clusters/charmed-k8s.mdx

```diff
@@ -0,0 +1,63 @@
+---
+description: Install Calico Enterprise on a Charmed Kubernetes cluster.
+---
+
+import InstallCharmedK8s from '@site/calico-enterprise/_includes/components/InstallCharmedK8s';
+
+# Charmed Kubernetes
+
+## Big picture
+
+Install $[prodname] on a Canonical Charmed Kubernetes cluster deployed with Juju.
+
+## Before you begin
+
+**CNI support**
+
+- Calico CNI for networking with $[prodname] network policy
+
+  The geeky details of what you get by default:
+
+  <GeekDetails
+    prodname='$[prodname]'
+    details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:IPIP,Routing:BGP,Datastore:Kubernetes'
+  />
+
+:::note
+
+By default, Charmed Kubernetes uses Calico for networking. Other CNI options like Flannel or Canal can be configured using bundle overlays during deployment.
+
+:::
+
+**Required**
+
+- A [compatible Charmed Kubernetes cluster](../compatibility.mdx#charmed-kubernetes)
+- Your cluster meets the [system requirements](requirements.mdx)
+- A [Tigera license key and credentials](calico-enterprise.mdx)
+- [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on your workstation
+- [Install juju](https://juju.is/docs/juju/install-juju) on your workstation (if managing the cluster directly)
+
+**Recommended**
+
+- Familiarity with [Juju concepts](https://juju.is/docs/juju) and [Charmed Kubernetes operations](https://ubuntu.com/kubernetes/charmed-k8s)
```

Comment:
Strike this. If we're assuming that people show up with a compatible Charmed Kubernetes cluster, then this is out of scope.